### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by cmg137**
Updated Microphone read function to interpret peak-to-peak voltage

Should help avoid error readings from the "Hail to Pitt" user input.

### CODE REVIEW

File: pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino [hunk 1]
```diff
@@ -0,0 +1,275 @@
+/**************************************************************
+ * AUTHOR(S):    Morgan Visnesky
```
Comment (vism2889) [inline pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino:2]: Add your name to the list of authors @cmg137

File: pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino [hunk 2]
```diff
@@ -0,0 +1,275 @@
+/**************************************************************
+ * AUTHOR(S):    Morgan Visnesky
+ * DATE:         03/18/2022
+ * FILENAME:     pitt-traditions.ino
```
Comment (vism2889) [inline pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino:4]: @cmg137 you had created a whole new file when making this change.  The file you added has an invalid file extension.

File: pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+  if (micVal > 2) 
+  {
+    cheer = true;
+  }
+  else 
+  {
+    cheer = false;
+  }
+
+  return cheer;
```
Comment (vism2889) [inline pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino:227]: @cmg137 indentation if off on lines 217 - 227.

File: pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+ */
+void displayMessage(String pMessage)
+{
+
+}
+
+/*
+ * Method to hold the functionality to read an analog value from the microphone
+ */
+boolean readFromMicrophone()
```
Comment (vism2889) [inline pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino:179]: @cmg137 can you add a link to where you found this online in the README file under Arduino citations & references section?

**Comment by cmg137** [comment]
I think I fixed the main issues, will double-check the indentation later. Fixed an issue with micval being assigned twice

**Pull request merged**